config/src/main/java/tech/pegasys/pantheon/config/GenesisConfigOptions.java

```diff
@@ -21,15 +21,15 @@
 
   boolean isIbft();
 
-  boolean isRevisedIbft();
+  boolean isIbft2();
```

Comment:
don't love that this is only 1 character different from isIbft

Reply:
Not sure what to do about that. The name is ibft 2. Thought perhaps could make the old version ibftLegacy in the method names?

Reply:
done. renamed old ibft methods to ibftLegacy to distinguish from ibft2